episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,847 @@
+
+:::::::::::::::::::::: testimonial
+
+### Early spread projections
+
+In the epidemic's initial phase, you can use `{epichains}` to apply a branching process model to project the number of future cases. Validation analysis showed that this is a rapid and low-detail model with forecast prediction intervals that are too certain ([Pearson et al., 2020](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.18.2000543#validationofthemodel-1)).
+
+We invite you to read the vignette on [Projecting infectious disease incidence: a COVID-19 example](https://epiverse-trace.github.io/epichains/articles/projecting_incidence.html)!
+
+::::::::::::::::::::::
+
+## Challenges
+
+:::::::::::::::::::::::::: challenge
+
+### Ebola's large outbreak potential
+
+Evaluate the potential of Ebola to generate explosive outbreaks. Use the appropriate package to access delay data from previous outbreaks.
+
+:::::::::::::: hint
+
+With `{epiparameter}`, you can access and use delay distributions from previous outbreaks.
+
+::::::::::::::
+
+:::::::::::::: solution
+
+```{r,message=FALSE,warning=FALSE}
+ebola_offspring <- epidist_db(
+  disease = "ebola",
+  epi_dist = "offspring",
+  single_epidist = TRUE
+)
+
+ebola_serial <- epidist_db(
+  disease = "ebola",
+  epi_dist = "serial interval",
+  single_epidist = TRUE
+)
+```
+
+<!-- [pending to complete] -->
+
+::::::::::::::
+
+:::::::::::::::::::::::::
+
+:::::::::::::::::::::::::: challenge
+
+### From a distribution of secondary cases
+
+[Christian Althaus, 2015](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(15)70135-0/fulltext) reused data published by [Faye et al., 2015 (Figure 2)](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(14)71075-8/fulltext#gr2) on the transmission tree on Ebola virus disease in Conakry, Guinea, 2014.
+
+Using the data under the **hint** tab, estimate the offspring distribution from the distribution of secondary cases. Then estimate the large outbreak potential from this data.
+
+::::::::::: hint
+
+Code with the transmission tree data written by [Christian Althaus, 2015](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(15)70135-0/fulltext):
+
+```{r,message=FALSE,warning=FALSE}
+# Number of individuals in the trees
+n <- 152
+# Number of secondary cases for all individuals
+c1 <- c(1, 2, 2, 5, 14, 1, 4, 4, 1, 3, 3, 8, 2, 1, 1,
+        4, 9, 9, 1, 1, 17, 2, 1, 1, 1, 4, 3, 3, 4, 2,
+        5, 1, 2, 2, 1, 9, 1, 3, 1, 2, 1, 1, 2)
+c0 <- c(c1, rep(0, n - length(c1)))
+
+c0 %>%
+  enframe() %>%
+  ggplot(aes(value)) +
+  geom_histogram()
+```
+
+:::::::::::
+
+::::::::::: solution
+
+<!-- [pending to complete] -->
+
+Remarkable, even with R0 less than 1
```

Comment:
```suggestion
Remarkably, even with R0 less than 1
```
Also, should this be one paragraph, rather than having a line break?

Reply:
completed and fixed in https://github.com/epiverse-trace/tutorials-middle/commit/60ceb9f801222656f8bc3ecaf7a72132096e3256